<div class="modal cancel-success-modal" id="return-order-failure-modal" tabindex="-1" role="dialog" aria-hidden="true">
  <div class="modal-dialog modal-dialog-centered" role="document">
    <div class="modal-content">
      <div class="modal-body">
        <div class="modal-inner">
          <span><img height="20" src="assets/images/close-icon1.svg" alt="right-check" class="fail-check"></span>
          <p *ngIf="orderType === orderTypes.CANCEL_ORDERS">
            Unable to cancel order no <em class="order-num">{{orderDetails?.order_number}}</em>
          </p>
          <p *ngIf="orderType === orderTypes.REATTEMPT_ORDERS">
            Unable to set order no <em class="order-num">{{orderDetails?.order_number}} as</em>
            <ng-container *ngIf="reattemptOrderStatus === reattemptOrderStatuses.MARK_AS_READY">
              marked as ready.
            </ng-container>
            <ng-container *ngIf="reattemptOrderStatus === reattemptOrderStatuses.MARK_AS_REATTEMPT">
              marked as reattempt.
            </ng-container>
          </p>
          <br>
          <p>Reason : {{cancelOrderFailure?.message}}</p>
        </div>
      </div>
    </div>
  </div>
</div>
